Sources/AWSLambdaEvents/APIGateway+V2+Encodable.swift

```diff
@@ -0,0 +1,50 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the SwiftAWSLambdaRuntime open source project
+//
+// Copyright (c) 2017-2022 Apple Inc. and the SwiftAWSLambdaRuntime project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+// See CONTRIBUTORS.txt for the list of SwiftAWSLambdaRuntime project authors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+import HTTPTypes
+
+#if canImport(FoundationEssentials)
+import class FoundationEssentials.JSONEncoder
+import struct FoundationEssentials.Data
+#else
+import class Foundation.JSONEncoder
+import struct Foundation.Data
+#endif
+
+public enum APIGatewayResponseError: Error {
+    case failedToEncodeBody(Error)
+}
+
+extension APIGatewayV2Response {
+
+    public init<Input: Encodable>(
+        statusCode: HTTPResponse.Status,
+        headers: HTTPHeaders? = nil,
+        body: Input,
+        isBase64Encoded: Bool? = nil,
```

Comment:
Given the JSONEncoder is returning a String is the base64 encoded option needed

Reply:
makes sense.  I removed that argument